package.json

```diff
@@ -18,6 +18,7 @@
     "deploy": "nx deploy",
     "test": "nx test",
     "lint": "nx workspace-lint && nx lint",
+    "tsc": "nx tsc --noEmit",
```

Comment:
What does this do?

Reply:
This was a leftover of debugging. Thought I had removed this, good spot.

Removed.